recipes/_install-linux.rb

```diff
@@ -0,0 +1,43 @@
+#
+# Cookbook Name:: datadog
+# Recipe:: _install-linux
+#
+# Copyright 2011-2015, Datadog
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Install the Apt/Yum repository if enabled
+include_recipe 'datadog::repository' if node['datadog']['installrepo']
+
+dd_agent_version = node['datadog']['agent_version']
+
+# If version specified and lower than 5.x
+if !dd_agent_version.nil? && dd_agent_version.split('.')[0].to_i < 5
```

Comment:
Let's add a `Chef::Log.deprecation("Support for Agent pre 5.x will be removed in in datadog cookbook version 3.0")` or something like that - so we can also remove anything related to `datadog-agent-base` in the future.